test/test-pages/gcba/platform-client.ts

```diff
@@ -0,0 +1,170 @@
+import { getCurrentUrlParams, getInputValue } from './utils';
+const client = (window as any).platformClient.ApiClient.instance;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9580329)
```suggestion
const client = (window as unknown).platformClient.ApiClient.instance;
```